replication/event.go

```diff
@@ -241,6 +241,9 @@ type QueryEvent struct {
 	StatusVars    []byte
 	Schema        []byte
 	Query         []byte
+
+	// in fact QueryEvent dosen't have the GTIDSet information, just for beneficial to use
```

Comment:
As `Gset` in `XID Event`,   `Query Event` ( DDL)  don't have `XID Event`,  so we need `Gset` to manage save point  by ourself